tutorial/index.md

```diff
@@ -56,15 +56,36 @@ At the end of this section, you should have a project that looks like this:
     main.lua
 ```
 
-## Drawing a ball
+Let's put something in `main.lua` and run our build script to make sure it all works. The minimum you need for a Playdate game to successfully build is the `playdate.update()` function, which is called right before every frame is rendered.
+
+Add this to `main.lua`:
+
+```lua
+function playdate.update()
+end
+```
 
-Now that you can see a blank screen in the simulator, let's make things a bit more interesting with our first sprite. We'll start with the ball.
+If you run the build script now, you'll get a blank screen in the simulator.
 
-As I mentioned in the intro, we're going to lean into the basic object-oriented programming (OOP) concepts offered by the Playdate SDK. These aren't strictly necessary and may have performance implications for larger games, but I find that they make code easier to reason about and extend.
+<!-- SCREENSHOT HERE -->
+
+Let's make things a bit more interesting with our first sprite! We'll start with the ball.
+
+
+## Drawing a ball
+
+As mentioned in the intro, we're going to lean into the basic object-oriented programming (OOP) concepts offered by the Playdate SDK. These aren't strictly necessary and may have performance implications for larger games, but I find that they make code easier to reason about and extend.
 
 Let's go ahead and make a simple `Ball` class in `main.lua` above the `playdate.update` function:
 
 ```lua
+-- import the OOP tools provided by the playdate SDK
+import "CoreLibs/object"
+-- import the playdate graphics library so we can draw things
+import "CoreLibs/graphics"
+-- import the playdate sprites library
+import "CoreLibs/sprites"
```

Comment:
Maybe worth a quick comment that these 3 imports will be used in basically any Playdate game you make.

Reply:
How's 3af5f5d30f30e2b84e0d0ecd73e154a5513bf3bf?